pkg/system/reconciler.go

```diff
@@ -284,9 +280,10 @@ func NewReconciler(
 	r.SecretServer.StringData["jwt"] = util.RandomBase64(16)
 	r.SecretServer.StringData["server_secret"] = util.RandomHex(4)
 
-	r.SecretDB.StringData["user"] = "noobaa"
-	r.SecretDB.StringData["password"] = util.RandomBase64(10)
-	r.SecretDB.StringData["db_name"] = "nbcore"
+	if r.NooBaa.Spec.ExternalPgSecret == nil {
+		r.SecretDB.StringData["user"] = "noobaa"
+		r.SecretDB.StringData["password"] = util.RandomBase64(10)
+	}
```

Comment:
why we don't need db name in postgres secret when we dont have the external string?

Reply:
it is provided inside the connection string. something like: `postgres://<user>:<password>@<host>:<port>/<<dbname>>`

Reply:
still not sure, when we Don't have ExternalPgSecret, don't we need it?

Reply:
ah no, we will use always nbcore